<launch>
  <param name="/cloud_to_map/resolution"  value="0.2"/>
  <param name="/cloud_to_map/search_radius" value="0.1"/>
  <node type="cloud_to_map_node" pkg="cloud_to_map" name="cloud_to_map" output="screen">
    <remap from="cloud" to="/sdf_map/occupancy" />
  </node>
  <node type="tf_broadcaster" pkg="hybrid_astar" name="tf_broadcaster" output="screen"/>
</launch>
